### PR TITLE
[ICDS] new design of login page for icds-cas

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @calellowitz @emord ,

I changed the login page for icds-cas server requested in this case: http://manage.dimagi.com/default.asp?281337

new_login_page

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/icds/templates/icds/login.html [hunk 1]
```diff
@@ -11,52 +11,38 @@
 
 
 {% block page_content %}
-{% initial_page_data 'implement_password_obfuscation' implement_password_obfuscation %}
+{#{% initial_page_data 'implement_password_obfuscation' implement_password_obfuscation %}#}
```
Comment (orangejenny) [inline custom/icds/templates/icds/login.html]: Why is this commented out?

File: custom/icds/templates/icds/login.html [hunk 2]
```diff
@@ -11,52 +11,38 @@
...
+            </a>
         </div>
     </div>
 
     <div class="row">
-        <div class="col-sm-1"></div>
-        <div class="col-lg-4">
-            <img style="height:300px" src="{% static 'icds/img/mobile.png' %}">
+        <div class="col-sm-2">
+            <img style="height:300px" src="{% static 'icds/img/phone_left.png' %}">
```
Comment (emord) [inline custom/icds/templates/icds/login.html]: why are there two phone pictures? they look the same to me

Comment (lwyszomi) [reply custom/icds/templates/icds/login.html]: @emord updated

File: custom/icds/templates/icds/login.html (hunk 1 above)
Comment (lwyszomi) [reply custom/icds/templates/icds/login.html]: @orangejenny updated

**Review by emord — APPROVED** [review]
looks fine to me. will let @orangejenny comment on her feedback

**Review by orangejenny — APPROVED** [review]

**Comment by lwyszomi** [comment]
Hi @emord @orangejenny @calellowitz 

I updated the login page for ICDS:
updated_login_page

Please let me know if you have any questions.

Regards,
Łukasz

**Review by orangejenny — COMMENTED** [review]
Minor: there's a `text-center` class you can use instead of the inline styles, which aren't good practice.

**Comment by lwyszomi** [comment]
Hi @orangejenny updated

**Review by emord — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]
thanks for updating

**Pull request merged**